src/frontend/src/functions/applicationperiodinfo.js

```diff
@@ -0,0 +1,149 @@
+import {
+    checkValidity,
+    getApplicationPeriodInfoJson,
+    getOngoingApplicationPeriodInfoJson,
+    getPersonalApplicationPeriodInfoJson,
+    getToken,
+    setApplicationPeriodInfoJson,
+    setOngoingApplicationPeriodInfoJson,
+    setPersonalApplicationPeriodInfoJson
+} from "./localstorage";
+import axios from "axios";
+import {url} from "./url";
+import {notification} from "antd";
+
+
+export async function submitApplicationPeriod(application_window_open, application_window_close,
+                                              applicable_periods, applicable_rooms, applicable_students) {
+    var data = JSON.stringify({
+        "application_window_open": application_window_open,
+        "application_window_close": application_window_close,
+        "applicable_periods": applicable_periods,
+        "applicable_rooms": ["string"],
+        "applicable_students": ["1004000", "1000000", "1003432", "1004515", "1004234", "justinHG"]
+    });
+    if (!checkValidity()) return undefined;
+    var config = {
+        method: 'post',
+        url: url + '/api/application_periods/',
+        headers: {
+            'accept': 'application/json',
+            'Authorization': 'Bearer ' + getToken(),
+            'Content-Type': 'application/json'
+        },
+        data: data
+
+    };
+    console.log(data);
+    await axios(config)
+        .then(function (response) {
+            console.log(JSON.stringify(response.data));
+            notification.success({
+                message: 'New Application Created Successfully',
+            });
+        })
+        .catch(function (error) {
+            console.log(error);
+            notification.error({
+                message: 'New Application Creation Failed',
+                description: 'Please try again'
+            });
+        });
+}
+
+export async function getAllApplicationPeriodInfo() {
+    if (!checkValidity) return undefined;
+    var application_data_json;
+    var config = {
+        method: 'get',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MarkHershey/SUTDHousingPortal/pullRequest?prid=7298734)
```suggestion
        method: "get",
```